File: functionoverloading.cpp
#include <iostream>

class Square {
public:
    void calculateArea(double side) {
        std::cout << "Area of the Square: " << side * side << std::endl;
    }

    void calculateArea(int side) {
        std::cout << "Area of the Square: " << side * side << std::endl;
    }
};

int main() {
    Square obj;
    obj.calculateArea(3.2);
    obj.calculateArea(10);
    return 0;
}

File: functionoverloading.java
class Square
{
void Ar( double side)
{
System.out.println("Area of the Square:"+side*side);
}
void Ar( float side)
{
System.out.println("Area of the Square:"+side*side);
}
}
class Area{
public static void main(String [] args)
{
Square obj = new Square();
obj.Ar(3.2);
obj.Ar(10);
}
}
